cfme/tests/infrastructure/test_infra_quota_via_services.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.automate.service_dialogs import ServiceDialog
+from cfme.services.catalogs.service_catalogs import ServiceCatalogs
+from cfme.services.catalogs.catalog import Catalog
+from cfme.services.catalogs.catalog_item import CatalogItem
+# from cfme.infrastructure.provider import InfraProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.configure.access_control import Tenant
+from cfme.services import requests
+from cfme.web_ui import flash
+from utils import testgen
+from utils.log import logger
+from utils.wait import wait_for
+
+
+pytestmark = [
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.usefixtures('vm_name', 'uses_infra_providers'),
+    test_requirements.quota
+]
+
+
+def pytest_generate_tests(metafunc):
```

Comment:
**Required**
Please check out ProviderFilter that @jkrocil added, linked is an example use with a specified class and flags, but you can specify required_fields as well.
https://github.com/ManageIQ/integration_tests/pull/4159/files#diff-12960d137dabebef95526e8ff8ebd4edR16